run-email.sh

```diff
@@ -0,0 +1 @@
+MONGO_URL=mongodb://localhost:27017/wekan MAIL_URL=smtp://noreplytest:noreply110@smtp.126.com:25/ MAIL_FROM=noreplytest@126.com meteor
```

Comment:
That's just an app password, no harm will be done, and it has been changed. I'll remove these 2 files from the diff